src/css/sections/reviews.css

```diff
@@ -0,0 +1,96 @@
+.review-cards-list {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+}
+
+@media only screen and (min-width: 767.99px) {
+    .review-cards-list {
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
+}
+
+.review-card {
+    background-color: var(--gray-color);
+    border-radius: 16px;
+    padding: 20px;
+}
+
+@media only screen and (min-width: 767.99px) {
+    .review-card {
+        max-width: 336px;
+    }
+}
+
+@media only screen and (min-width: 1152px) {
+    .review-card {
+        max-width: 320px;
+    }
+}
+
+.review-card:not(:first-child) {
+    display: none;
+}
+
+@media only screen and (min-width: 767.99px) {
+    .review-card:not(:first-child) {
+        display: inline-block;
+    }
+
+    .review-card:nth-child(n+3) {
+        display: none;
+    }
+}
+
+@media only screen and (min-width: 1152px) {
+    .review-card:not(:first-child) {
+        display: inline-block;
+    }
+
+    .review-card:nth-child(n+3) {
+        display: inline-block;
+    }
+}
+
+.review-header {
+    margin-bottom: 20px;
+    padding: 14px 14px;
+    border-radius: 8px;
+    background-color: var(--green-color);
+    display: flex;
+}
+
+.review-photo {
+    max-width: 50px;
```

Comment:
let's use the jpg versions of images, without rounding, and add rounding with border-radius and overflow.
JPG will take less size than png, so app will load faster.

![Uploading Screenshot 2024-10-03 at 02.08.26.png…]()

Reply:
How did you download image without rounding?